### ISSUE TITLE
Which file is for Mol-PCBA dataset?

### ISSUE THREAD

**Issue by ZhaoningYu1996** [lukecavabarrett/pna#10]
Hi,
Could you tell me where is main_PCBA file?
Thank you,
Zhaoning

**Comment by gcorso** [comment]
Hi, you can find the scripts on the [DGN repository](https://github.com/Saro00/DGN) where you should be able to run the following command to get the leaderboard results for PNA:

`python main_PCBA.py --type_net="complex" --batch_size=512 --lap_norm="none" --weight_decay=3e-6 --L=4 --hidden_dim=510 --out_dim=510 --residual=True --edge_feat=True  --readout=sum --graph_norm=True --batch_norm=True --aggregators="mean sum max" --scalers="identity" --config "configs/molecules_graph_classification_DGN_PCBA.json"  --lr_schedule_patience=4 --towers=5 --dropout=0.2 --init_lr=0.0005 --min_lr=0.00002 --edge_dim=16 --lr_reduce_factor=0.8
`

**Comment by ZhaoningYu1996** [comment]
Are PNA and DGN sharing the same code? Do they have the same files and same hyperparameters?
I found the name of the config file for PNA is ''**_DGN_**'', so I am a little confused.

**Comment by gcorso** [comment]
Yes, simply different aggregators
